evals/agent_evals/testing_constants.py

```diff
@@ -0,0 +1,4 @@
+FILES_SECTION_HEADING = "Files in the current directory:"
```

Comment:
I think thisis duplicate with `evals.prompts.constants`. Let's use those so we minimize how many places we need to keep in sync.